fractal/cmd/_workflow.py

```diff
@@ -207,10 +207,16 @@ async def patch_workflow(
     *,
     project_id: int,
     workflow_id: int,
-    **workflow_update_dict,
+    new_name: Optional[str] = None,
+    **update,
```

Comment:
Is there any possible way to have `update != {}`?
If not, let's get rid of it (ref #460).